services/web/server/src/simcore_service_webserver/catalog/_handlers.py

```diff
@@ -74,7 +74,7 @@ def create(cls, request: Request) -> "_RequestContext":
 
 
 class _ServicePathParams(BaseModel):
-    service_key: ServiceKeyEncoded
+    service_key: ServiceKey
```

Comment:
As expected, this change will break the tests. 
I think you were right when claimed that our test client do not represent the actual client calls from the front-end

A fast way out of this issue (until we have the time to refactor it properly) is to define here ``service_key: str``  which will "disable" the validation at the handler level. This is effectively as it was before (i.e. no validation whatsoever!) ...

Reply:
ok I fixed it by always unquoting. this makes the tests pass and does not break the production neither. I think this must be something weird with the aiohttp_server fixture??